index.html

```diff
@@ -5,23 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Please add all unnecessary files and folders like `vscode` to `gitignore`.

Reply:
Done